src/Runner/TestSuiteSorter.php

```diff
@@ -79,6 +141,38 @@ private function randomize(array $tests): array
         return $tests;
     }
 
+    private function sortDefectsFirst(array $tests): array
+    {
+        \usort($tests, [self::class, 'cmpDefectPriorityAndTime']);
```

Comment:
please avoid calling methods by their names. Static Code Analysis will not detect that...

Reply:
Fixed, thanks for the syntax suggestion :)